packages/manager/modules/pci/src/projects/project/kubernetes/details/nodes/billing-type/billing-type.routing.js

```diff
@@ -0,0 +1,23 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider
+    .state('pci.projects.project.kubernetes.details.nodes.billing-type', {
+      url: '/billing-type?nodeId',
+      views: {
+        modal: {
+          component: 'pciProjectKubernetesNodesBillingTypeComponent',
+        },
+      },
+      layout: 'modal',
+      params: {
+        nodeId: null,
+        nodeName: null,
+        instanceId: null,
+      },
+      resolve: {
+        goBack: /* @ngInject */ goToKubernetesNodes => goToKubernetesNodes,
+        nodeId: /* @ngInject */ $stateParams => $stateParams.nodeId,
```

Comment:
Don't use `$stateParams`, it's deprecated (https://ui-router.github.io/ng1/docs/latest/modules/injectables.html#_stateparams), use `$transition$` instead (https://ui-router.github.io/ng1/docs/latest/modules/injectables.html#_transition_)